pkg/util/clusteroperator/status.go

```diff
@@ -35,31 +40,52 @@ func SetStatusCondition(oldConditions []configv1.ClusterOperatorStatusCondition,
 	return newConditions
 }
 
-// ConditionsEqual returns true if and only if the provided slices of conditions
-// (ignoring LastTransitionTime) are equal.
-func ConditionsEqual(oldConditions, newConditions []configv1.ClusterOperatorStatusCondition) bool {
-	if len(newConditions) != len(oldConditions) {
-		return false
+// ElementsEqual return true if listX items match listY items ignoring the order of the items.
+// In case of duplicate items, the number of appearances of each item in both lists should match.
+// listX and listY must be an array or slice.
+func ElementsEqual(listX, listY interface{}, opts []cmp.Option) (bool, error) {
```

Comment:
What do you think about putting this in `pkg/util`? I guess tests could go there too.